docs/user/rstudio/ide/guide/code/debugging.qmd

```diff
@@ -233,7 +236,7 @@ For this reason, breakpoints in Shiny applications *only* work inside the `shiny
 
```

Comment:
typo in "or other .R sources used in Shiny **applcations**. This may be improved in a future version of RStudio."